cmd/kubeadm/app/preflight/checks.go

```diff
@@ -431,6 +431,19 @@ func (kubever KubernetesVersionCheck) Check() (warnings, errors []error) {
 	return nil, nil
 }
 
+type KubeletVersionCheck struct{}
```

Comment:
This should have comments, we now enforce golint :)

Reply:
ok. will add.